content/go/concepts/switch/switch.md

```diff
@@ -0,0 +1,120 @@
+---
+Title: 'Switch'
+Description: 'A switch statement allows the selective execution of one of several blocks of code based on the value of an expression.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Comparison'
+  - 'Control Flow'
+  - 'Switch'
+CatalogContent:
+  - 'learn-go'
+  - 'paths/back-end-engineer-career-path'
+---
+
+A `switch` statement allows the selective execution of one of several blocks of code based on the value of an expression. It can be thought of as an expanded `if ... then ... else` statement, with many potential `then` clauses.
```

Comment:
Are we sure it's an `if...then...else` statement and not just `if..else`? I am searching the official Go docs and can't find that specific reference: https://go.dev/ref/spec#If_statements

Reply:
I think you're right @Dusch4593 . They compare it to an `if...else` statement on their "A Tour of Go" webpage: https://go.dev/tour/flowcontrol/9#:~:text=A%20switch%20statement%20is%20a,all%20the%20cases%20that%20follow.